build.gradle.kts

```
@@ -21,6 +21,7 @@ repositories {
 extra["vaadinVersion"] = "14.3.3"
 
 dependencies {
+    implementation("org.springframework.boot:spring-boot-starter-mail")
     implementation("org.springframework.boot:spring-boot-starter-data-jpa")
```

같은 그룹 내에서 이름으로 정렬하는 것은 어떨까요?

```suggestion
    implementation("org.springframework.boot:spring-boot-starter-data-jpa")
    implementation("org.springframework.boot:spring-boot-starter-mail")
```

---

좋은 의견 감사합니다.
나중에 의존성 많아지면 보는거나 찾는것도 편하겠네요